src/modules/px4iofirmware/mixer.cpp

```diff
@@ -533,30 +579,19 @@ mixer_handle_text(const void *buffer, size_t length)
 			return 0;
 		}
 
+		/* check if the last item has been processed - bail out if not */
+		if (mixer_update_pending) {
+			return 1;
```

Comment:
Can you add the same comment here as above (`the caller is expected to retry`)? Otherwise one could think that the potential race here between `mixer_tick` and multiple interrupts can be a problem.